README.md

```diff
@@ -89,7 +104,31 @@ entities (no need to draw an ERD):
 
 Your answer:
 ```
-Replace this with your answer
+An ERD is a tool used to visualize and describe the date relating to the major entities that exist in a program.
+
+A genie and a lamp have a one-to-one relationship, because there can only be one genie to a lamp and a lamp can only have one genie.
+ - Genie
+    - name #string
+    - confined_to_lamp #date
+    - location #string
+ - Lamp
+    - last_rubbed #date
+    - location #string
+    - model #string
+    - last_master #string
+
+A person and a pet is a one-to-many relationship, because a person can have many pets, but a pet can only have one person.
+ - Person
+    - name #string
+    - birthdate #date
+    - age #integer
+    - sex #string
+ - Pet
+    - name #string
+    - birthdate #date
+    - age #integer
+    - fave_food #string
+
```

Comment:
again, how does the erd change if we want to incorporate an association?Maybe a foreign key somewhere.